### ISSUE TITLE
MddBootstrapAutoInitializer does not compile with enabled ImplicitUsings

### ISSUE THREAD

**Issue by HavenDV** [microsoft/WindowsAppSDK#1686]
Bug with the intersection of Microsoft.WindowsAppSDK.Runtime.Version with System.Version.

It needs to use `global::` prefixes.

Env:
`Microsoft.WindowsAppSDK`: `1.0.0-preview3`
`WindowsPackageType`: `None`

**Comment by HavenDV** [comment]
Workaround: Create `Fix.cs` with this content
```cs
// Workaround for https://github.com/microsoft/WindowsAppSDK/issues/1686
global using Version = Microsoft.WindowsAppSDK.Runtime.Fix.Version;

namespace Microsoft.WindowsAppSDK.Runtime.Fix;

public class Version
{
    public const ushort Major = 0;
    public const ushort Minor = 297;
    public const ushort Build = 2018;
    public const ushort Revision = 0;
    public const ulong UInt64 = 0x0000012907E20000;
    public const string DotQuadString = "0.297.2018.0";
}
```

**Comment by riverar** [comment]
Am confused. You turned on _implicit usings_ which brought together two disparate namespaces and a type collided (as expected). How is this Windows App SDK's fault?

**Comment by HavenDV** [comment]
It adds the file to compile like a generator.

**Comment by HavenDV** [comment]
Please see PR

**Comment by riverar** [comment]
I'm not confused about the changes you made. I'm just trying to understand what problem _implicit usings_ solves and why you think Windows App SDK should support this.

**Comment by HavenDV** [comment]
As far as I know, they will be enabled by default for new projects.

**Comment by riverar** [comment]
That decision appears to have been reversed. https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/implicit-namespaces-rc1
> As part of the initial templates update, we enabled implicit usings by default (AKA opt-out) for .NET 6 (net6.0) projects (including if you updated an app from .NET 5 to .NET 6). That has been changed. We’ve updated the SDK so that all the new features are opt-in.

**Comment by HavenDV** [comment]
I mean, not at the SDK level, but when creating a new project, the MSBuild property will be set to true in the csproj file.

**Comment by HavenDV** [comment]
It seems it was not included in the 1.0.0 stable release. I still get an error if I don't apply the workaround

**Comment by DrusTheAxe** [comment]
https://task.ms/38083595 for 1.0-servicing
